web/packages/teleport/src/services/websession/websession.ts

```diff
@@ -146,7 +146,34 @@ const session = {
       return false;
     }
 
-    // Renew session if token expiry time is less than 3 minutes.
+    const token = this._getBearerToken();
+    var expiresIn, sessionExpiresIn;
+    if (token) {
+      // Convert seconds to millis.
+      if (token.expiresIn) {
+        expiresIn = token.expiresIn * 1000;
+      }
+      if (token.sessionExpiresIn) {
+        sessionExpiresIn = token.sessionExpiresIn * 1000;
+      }
+    }
```

Comment:
we should probably just early return if token doesnt exist

also, we can use some operators to set a "default value" for these
```suggestion
   if (!token) {
     return false
   }
   const expiresIn = (token.expiresIn ?? 0) * 1000
   const sessionExpiresIn = (token.sessionExpiresIn ?? 0) * 1000
```
(typed in github editor so maybe a typo or two but you get the idea)